packages/core/database/lib/index.js

```diff
@@ -20,6 +20,7 @@ class Database {
       connection: {},
       settings: {
         forceMigration: true,
```

Comment:
You will need to add the default runMigrations: true here then :)

Reply:
yes! good catch, thanks